gatsby/src/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,59 @@
+import { Link, graphql, useStaticQuery } from "gatsby";
+import * as React from "react";
+
+import styles from "./Hero.module.scss";
+
+export type LastUpdatedDataQueryResult = {
+	bnfMetadata: {
+		lastUpdatedDate: string;
+		lastUpdatedDateFormatted: string;
+		runTag: string;
+	};
+};
+
+type HeroProps = {
+	isBNF: boolean;
```

Comment:
It seems odd here that `isBNF` when we have the hook for it - that's what we use everywhere else, directly in the component